Domains/Smartlighting/LightSchedules.md

```diff
@@ -0,0 +1,286 @@
+
+For `ActionTime` values `SUNRISE` or `SUNSET` the value of `TriggerType` specifies what the actual
+switching time should be.
+
+| **`TriggerType`** | **Description**                                                                                           |
+| -----------------:| --------------------------------------------------------------------------------------------------------- |
+| `LIGHT_TRIGGER`   | [astronomical time with sensor input](#astronomical-time-with-sensor) determine the actual switching time |
+| `ASTRONOMICAL`    | the [calculated astronomical time](#astronomical-time) for sunrise or sunset is the switching time        |
+
+### Fixed Time
+
+For `ActionTime` `ABSOLUTETIME` a fixed time can be set for the switching moment as `Time`.<br>
+The `Time` value needs to be formatted in a way the [protocol implementations](#protocol-implementations-for-light-schedules)
+can handle. For the currently listed implementations, you should be fine when you use a format from:
+- `hh:mm`;
+- `hh:mm:ss`;
+- `hh:mm:ss.SSS`
+
+With `hh` from `00` to `23`, `mm` from `00` to `59`, `ss` from `00` to `59` and `SSS` from `000` to `999`.<br>
+Some protocols may accept more precise time formats than they support. The IEC61850 implementation for
+instance, will silently apply only the hours and minutes from any of the formats listed above.
+
+### Astronomical Time
+
+For `ActionTime` `SUNRISE` or `SUNSET` with `TriggerType` `ASTRONOMICAL` the calculated astronomical
+sunrise or sunset time (by the switching device, based on its longitude and latitude) will be used to
+determine the switching moment.
+
+#### Astronomical Offsets
+
+If an astronomical offset is configured, it has to be added to the calculated astronomical time to
+determine the time to be used as the switching moment.<br>
+For positive offset values, the astronomical time for the switching moment will be the configured
+amount of minutes after the calculated astronomical sunrise or sunset time, while for negative values
+the astronomical time used will be the number of minutes before the calculated astronomical sunrise
+or sunset time.
+
+**Astronomical Sunrise Offset**
+
+The astronomical sunrise offset is applied with entries with `ActionTime` `SUNRISE` and
+`TriggerType` `ASTRONOMICAL`.<br>
+The following picture is an example of switching off at 07:30; the calculated astronomical sunrise
+(say at 07:15 for the day shown) plus 15 minutes (configured as `AstronomicalSunriseOffset` `15`).
+
+![Switching off at astronomical sunrise with 15 minutes offset](./astronomical-time-sunrise-with-offset.png)
+
+**Astronomical Sunset Offset**
+
+The astronomical sunset offset is similar to the [astronomical sunrise offset](#astronomical-sunrise-offset),
+except that it is applied with entries with `ActionTime` `SUNSET` and `TriggerType` `ASTRONOMICAL`.
+
+### Astronomical Time With Sensor
+
+For `ActionTime` `SUNRISE` or `SUNSET` with `TriggerType` `LIGHT_TRIGGER` the calculated astronomical
```

Comment:
I would make it: For `ActionTime` `SUNRISE` or `SUNSET` with `TriggerType` `LIGHT_TRIGGER` a light trigger will be used if within the trigger window around the calculated astronomical time to determine the switching moment.